src/main.py

```diff
@@ -127,8 +162,93 @@ async def on_member_update(self, prev, cur):
         if cur.activity.type is discord.ActivityType.playing:
             user.add_games([cur.activity.name])
             logging.info(
-                f'User starting playing {cur.activity.name}. Added to gamelist')
-
+                f'User starting playing {cur.activity.name}. Added to user gamelist')
+            self.add_games_to_gamelist(cur.activity.name)
+
+    def add_games_to_gamelist(self, games):
```

Comment:
You should require `games` to always be a list. If it's a string, it can also just be a list of 1 game.